scripts/build_plugin_docker_ubuntu.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
 
+
+# Usage: 
+# $1: docker image to use (tested with ubuntu:18.04 and ubuntu:20.04)
+# $2 path to local pkg folder containing both trelis/cubit install package and sdk
```

Comment:
Any reason we can't automatically download the install packages and SDK from Box like we do in the GitHub actions?

Reply:
yes we don't want to publicly expose the URL

Reply:
Coreform may be moving towards a direct download  (per Shimwell)

Reply:
That would be great. Let's hope for that soon.